CMakeLists.txt

```diff
@@ -332,11 +332,24 @@ if (CMAKE_SYSTEM_NAME MATCHES "Darwin") # we must call library install here in
             ARCHIVE DESTINATION lib)
     endforeach ()
 
+endif ()
+
+# Copy pcm files
+if (CMAKE_SYSTEM_NAME MATCHES "Darwin" OR CMAKE_SYSTEM_NAME MATCHES "Windows")
+
     # Copy pcm files to bin
+    file(GLOB PCMFILES "${CMAKE_CURRENT_BINARY_DIR}/rootdict/*.pcm")
```

Comment:
The PCM files are produced during compilation. Is it OK to find them with this code code? Because it seems to be executed during cmake, not make

Reply:
Yes, this might be the issue but still annoying because I cannot reproduce locally, I have moved to `install( DIRECTORY`